js/containers/buttons/Bypass.jsx

```diff
@@ -32,8 +32,7 @@ const mapStateToProps = function(state) {
 const mapDispatchToProps = function(dispatch) {
 	return {
 		onClick: function() {
-			dispatch(actions.createDeployment())
```

Comment:
because it's called form the continue button "between" deploy plan and the step buttom